### DISCUSSION TITLE
performances and sysex on a usb pen

### DISCUSSION THREAD

**Discussion by Piattica** [probonopd/MiniDexed#577] [Q&A]
would it be possible to put performances and sysex on a usb pen
it would be easier to update and mantain...

**Comment by probonopd** [comment]
We need to prioritize development resources and manage code complexity. And not all Raspberry Pi models that MiniDexed supports have more than one USB port.

So I don't think it is a priority at the moment, as it would increase hardware requirements and complexity.

Maybe we should implement sending files over MIDI Sysex messages to save them into files?
